helm/qontract-reconcile/templates/template.yaml

```diff
@@ -67,20 +79,32 @@ objects:
             </filter>
 
             <match integration>
-              @type slack
-              webhook_url ${SLACK_WEBHOOK_URL}
-              channel ${SLACK_CHANNEL}
-              icon_emoji ${SLACK_ICON_EMOJI}
-              username sd-app-sre-bot
-              flush_interval 10s
-              message "\`\`\`[{{ $integration.name }}] %s\`\`\`"
+              @type copy
+              {{- if $integration.logs.slack }}
+              <store>
+                @type slack
+                webhook_url ${SLACK_WEBHOOK_URL}
+                channel ${SLACK_CHANNEL}
+                icon_emoji ${SLACK_ICON_EMOJI}
+                username sd-app-sre-bot
+                flush_interval 10s
+                message "\`\`\`[{{ $integration.name }}] %s\`\`\`"
+              </store>
+              {{- end }}
+              {{- if $integration.logs.cloudwatch }}
+              <store>
+                @type cloudwatch_logs
+                log_group_name ${LOG_GROUP_NAME}
+                log_stream_name ${LOG_STREAM_NAME}
```

Comment:
You can use integration.name here if it makes sense

Reply:
This approach will create a unique stream for each integration for each run instead of putting logs from various instances in same stream. Everytime pod is restarted, we get a new stream. 

You can still search across streams using CloudWatch.

Reply:
Stream name will be set to the pod name. Pod name will tell us which integration and which instance of it wrote those logs.

Reply:
Why would it create a log stream per run?

Reply:
I don't think that setting the log stream name to the pod name is desired.

I meant to use `{{ $integration.name }}`

Reply:
Why not?

Reply:
Each pod has a different name. We want to see a stream of events from each integration over time. No?

Reply:
Yes and for that you don't worry about stream name. You can search on a log group in CloudWatch.